### DISCUSSION TITLE
inch to cm pound to kg in post titles

### DISCUSSION THREAD

**Discussion by Samet1771** [Docile-Alligator/Infinity-For-Reddit#1070] [Not planned Ideas] [answered]
It can be very useful

**Comment by Wladefant** [comment, marked as answer]
I will answer with the same text the Apollo developer wrote:

This is really cool on paper, but as others said I wonder the real world practicality of it. For converting between lbs and kg (imperial and metric) 100%, wouldn't be too hard, but in my experiences 95% of the time there's a bot underneath that replied with the conversion, so it's not super helpful. Same with Fahrenheit and Celsius.

The rest are cool but VERY difficult to do as others mentioned as they're fraught with edge cases. Two letter abbreviations can have MANY different things they are short for, rather than just city codes, and even things like saying 1" to mean 1 inch could be present in a quote for instance, where someone says "I think my favorite number is 1". Parsing out all these edge cases and trying to compute them, as someone said, would be closer to a machine learning task that Google or Apple would compute, and would be very, very difficult to do on the fly.

**Comment by Wladefant** [comment]
Other opinions: https://www.reddit.com/r/apolloapp/comments/rj3hfs/are_you_also_annoyed_when_you_dont_understand/
